short-survey-checklist.md

```diff
@@ -0,0 +1,21 @@
+# Checklist for MDN short surveys
+
+This document lists the standard operating procedure for creating MDN short surveys. 
+
+# Mandatory fields
```

Comment:
I realize these comes from the MDN short survey process, but suggestions below to improve a bit from there